manifester_settings.yaml.example

```diff
@@ -0,0 +1,36 @@
+#rhsm-manifester settings
```

Comment:
Prefix all manifester settings with `MANIFESTER` key like we have it in the rest of the robottelo configuration. This might require a change on manifester side.

Reply:
Now I see https://github.com/SatelliteQE/manifester/pull/11. So I might be totally off with the above suggestion.